rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/snapshot.get/10_basic.yml

```diff
@@ -195,12 +195,17 @@ setup:
       snapshot.get:
         repository: test_repo_get_1
         snapshot: test_snapshot_with_metadata
+        human: true
 
   - is_true: responses.0.snapshots
   - match: { responses.0.snapshots.0.snapshot: test_snapshot_with_metadata }
   - match: { responses.0.snapshots.0.state: SUCCESS }
   - match: { responses.0.snapshots.0.metadata.taken_by: test }
   - match: { responses.0.snapshots.0.metadata.foo.bar: baz }
+  - gt:  { responses.0.snapshots.0.index_details.test_index.shard_count: 0 }
```

Comment:
uff :) Do we really want to add this information here? I'd almost rather not do that unless we default to `false` for `verbose` in `8.x`. The size of the get snapshots response is already a problem today at times (in Cloud and debugging in general) and this will massively increase its size for large index counts.

Reply:
Not sure really, I think it's useful, but I'd be happy to add another query parameter to control it.

Reply:
Yea I'd say lets add another parameter and ++ it's definitely an interesting thing to expose to users. (Maybe have it default to `false` since these responses are already problematic in size?)

Reply:
Ok, made optional in 1e8692d7c9957c3708ada4e9fd1c1fa8794b4fd0.